core/src/main/java/tc/oc/pgm/commands/AdminCommands.java

```diff
@@ -12,23 +12,30 @@
 import javax.annotation.Nullable;
 import net.kyori.text.TranslatableComponent;
 import net.kyori.text.format.TextColor;
-import org.bukkit.ChatColor;
+import net.md_5.bungee.api.ChatColor;
 import org.bukkit.command.CommandSender;
 import tc.oc.pgm.api.PGM;
 import tc.oc.pgm.api.Permissions;
 import tc.oc.pgm.api.map.MapInfo;
 import tc.oc.pgm.api.map.MapOrder;
 import tc.oc.pgm.api.match.Match;
-import tc.oc.pgm.api.match.MatchManager;
 import tc.oc.pgm.api.party.Competitor;
+import tc.oc.pgm.community.commands.ModerationCommands;
```

Comment:
Based on the wording in #326 i do not think PGM should depend on the `community` module being present.